thirdparty/openjdk-cacerts/zimbra-openjdk-cacerts/rpm/SPECS/openjdk-cacerts.spec

```diff
@@ -1,7 +1,7 @@
 Summary:            CA Certs keystore for OpenJDK
 Name:               zimbra-openjdk-cacerts
-Version:            1.0.8
-Release:            ITERATIONZAPPEND
+Version:            1.0.9
+Release:            1zimbra8.7b1ZAPPEND
```

Comment:
The package should probably have a dynamic version or release number, since its contents may change with each build now.  But I don't see a useful version number to derive from the downloaded certdata.txt file.  Maybe just a current year+month?